integration-tests/tests/RegisterAddress.spec.ts

```diff
@@ -0,0 +1,62 @@
+// [object Object]
+// SPDX-License-Identifier: Apache-2.0
+
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api';
+
+const CREDO_PER_CTC = 1_000_000_000_000_000_000;
+const POINT_01_CTC = CREDO_PER_CTC / 100;
+
+describe('RegisterAddress', (): void => {
+    let api;
+    let alice;
+
+    beforeEach(async () => {
+        process.env.NODE_ENV = 'test';
+
+        const provider = new WsProvider('ws://127.0.0.1:9944');
+
+        api = await ApiPromise.create({ provider });
+
+        const keyring = new Keyring({ type: `sr25519` });
+        alice = keyring.addFromUri('//Alice', { name: 'Alice' });
+    });
+
+    afterEach(async () => {
+        await api.disconnect();
+    });
+
+    it('fee is min 0.01 CTC', async () => {
+        expect.assertions(2);
+
+        const unsubscribe = await api.tx.creditcoin
+            .registerAddress('Ethereum', '0x3C6a6762f969B36bb1a6DBD598A5DC9800284D77')
+            .signAndSend(alice, ({ status, events, dispatchError }) => {
```

Comment:
@jfrank-summit, @nathanwhit this test is modeled after the original example from Nathan but I am having some troubles with it. 

The expectation on line 29 will fail b/c jest finishes very quickly. We get

```bash
$ yarn test tests/RegisterAddress.spec.ts 
yarn run v1.22.17
$ polkadot-dev-run-test --forceExit --runInBand tests/RegisterAddress.spec.ts
$ polkadot-dev-run-test --forceExit --runInBand tests/RegisterAddress.spec.ts
  console.warn
    2022-03-29 19:51:51        API/INIT: RPC methods not decorated: creditcoin_eventsSubscribe, creditcoin_eventsUnsubscribe, creditcoin_getEvents

      at apply (node_modules/@polkadot/util/logger.cjs:81:22)
      at Object.warn (node_modules/@polkadot/util/logger.cjs:161:14)
      at ApiPromise._filterRpcMethods (node_modules/@polkadot/api/base/Decorate.cjs:311:9)
      at ApiPromise._filterRpc (node_modules/@polkadot/api/base/Decorate.cjs:274:10)
      at ApiPromise._metaFromChain (node_modules/@polkadot/api/base/Init.cjs:350:10)
      at ApiPromise._loadMeta (node_modules/@polkadot/api/base/Init.cjs:278:229)

 FAIL  tests/RegisterAddress.spec.ts (6.031 s)
  RegisterAddress
    ✕ fee is min 0.01 CTC (1363 ms)

  ● RegisterAddress › fee is min 0.01 CTC

    expect.assertions(2)

    Expected two assertions to be called but received zero assertion calls.

      27 |
      28 |     it('fee is min 0.01 CTC', async () => {
    > 29 |         expect.assertions(2);
         |                ^
      30 |
      31 |         const unsubscribe = await api.tx.creditcoin
      32 |             .registerAddress('Ethereum', '0x3C6a6762f969B36bb1a6DBD598A5DC9800284D77')

      at Object.<anonymous> (tests/RegisterAddress.spec.ts:29:16)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        6.117 s
Ran all test suites matching /tests\/RegisterAddress.spec.ts/i.
Force exiting Jest: Have you considered using `--detectOpenHandles` to detect async operations that kept running after all tests finished?
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I am also following jest example [here](https://jestjs.io/docs/tutorial-async#asyncawait). 

I don't see any of the `console.log` statements and at this point I'm not sure why.

Reply:
In the CI logs and also sometimes locally there's this message:

```bash
Cannot log after tests are done. Did you forget to wait for something async in your test?
    Attempted to log "******* STATUS=Ready".
```

However from what I can tell the code is waiting appropriately for the async operation.